concordium-sdk/src/main/java/com/concordium/sdk/responses/accountinfo/credential/AttributeType.java

```diff
@@ -0,0 +1,34 @@
+package com.concordium.sdk.responses.accountinfo.credential;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public enum AttributeType {
```

Comment:
Please add a short doc at the enum declaration about what the `AttributeType` is reflecting.

Reply:
Can you add a short doc here and then it's good to go. Thanks